ui/src/containers/CreateVolume.js

```diff
@@ -734,6 +755,49 @@ const CreateVolume = (props) => {
                         error={touched.path && errors.path}
                         onBlur={handleOnBlur}
                       />
+                    ) : (
+                      <>
+                        <SizeFieldContainer>
+                          <Input
+                            name="sizeInput"
+                            type="number"
+                            min="1"
+                            value={values.sizeInput}
+                            onChange={handleChange('sizeInput')}
+                            label={`${intl.translate('volume_size')}*`}
+                            error={touched.sizeInput && errors.sizeInput}
+                            onBlur={handleOnBlur}
+                          />
+                          <SizeUnitFieldSelectContainer>
+                            <Input
+                              clearable={false}
+                              type="select"
+                              options={optionsSizeUnits}
+                              noOptionsMessage={() =>
+                                intl.translate('no_results')
+                              }
+                              name="selectedUnit"
+                              onChange={handleSelectChange('selectedUnit')}
+                              value={getSelectedObjectItem(
+                                optionsSizeUnits,
+                                values?.selectedUnit,
+                              )}
+                              error={
+                                touched.selectedUnit && errors.selectedUnit
+                              }
+                              onBlur={handleOnBlur}
+                            />
+                          </SizeUnitFieldSelectContainer>
+                        </SizeFieldContainer>
+                        <Input
+                          name="vgName"
+                          value={values.vgName}
+                          onChange={handleChange('vgName')}
+                          label={`Volume Group Name*`}
+                          error={touched.vgName && errors.vgName}
+                          onBlur={handleOnBlur}
+                        />
+                      </>
```

Comment:
Maybe we can extract the fields to a <LVMAdditonnalFields /> component ? It would then make sense to validate those additional fields in that component and just add an onChange handler prop to that component that just addup the fields value to the submitted object.

Reply:
Hi @JBWatenbergScality 
I am not sure of the benefit to extract these fields. For me, it would add some complexity than keep everything centralized. ( but I am open to discuss).

However, I see a reusable combined field  `sizeInputCombinedField` which is used in both `SPARSE_LOOP_DEVICE` and `LVM_LOGICAL_VOLUME`
https://github.com/scality/metalk8s/pull/3410/commits/8cd3fda6d7454bab3bcacfc926a20f0cb592c808